3party/opening_hours/opening_hours.hpp

```diff
@@ -133,6 +134,7 @@ class Time
   void SetEvent(TimeEvent const & event);
 
   HourMinutes const & GetHourMinutes() const;
+  HourMinutes & GetHourMinutes();
```

Comment:
Or, just rename this method do HourMinutes() and get rid of {Get|Set}HourMinutes.